docs/roadmap/cosmos-hub-roadmap-2.0.md

```diff
@@ -32,9 +32,19 @@ The upgrades aim to add features such as liquidity, economic security, usability
   - Transfer ATOM, ETH, ERC-20, and other Cosmos tokens between Ethereum and the Gravity Bridge Chain and by extension all IBC connected chains.
   - Fee and reward model hosted across Cosmos and Ethereum
 
-## Theta Upgrade (expected Q1 2022)
-
+## v7-Theta Upgrade (expected Q1 2022)
```

Comment:
```suggestion
## v07-Theta Upgrade (expected Q1 2022)
```

Reply:
why this?

Reply:
so that the titles of the upgrades will sort correctly in any scenario they're used as directory or filenames : )

Reply:
maybe over optimization in light of easier to read